dask_expr/_groupby.py

```diff
@@ -1034,7 +1034,7 @@ def __init__(
 
         self.obj = obj[projection] if projection is not None else obj
         self.sort = sort
-        self.observed = observed
+        self.observed = observed if observed is not None else False
```

Comment:
Ah - Looks like cudf uses an `observed=True` default, and doesn't support `observed=False`, so this breaks things a bit with cudf. Do you remember the dask/dask failure this was meant to fix? (I don't think I see failures in dask-expr when I change this back)

Reply:
You'll only see this with a pandas version that has the observed deprecation generally speaking and I indeed think that there is no test in dask-expr, only over in dask/dask.

IIRC we were showing the deprecation warning multiple times without this change